From 96f514f9d92d22f5f8030926b5165db13a0739fc Mon Sep 17 00:00:00 2001
From: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
Date: Mon, 15 May 2023 10:54:16 -0300
Subject: [PATCH] csm-application-mobility: update dependencies

This patch consists of removing the cert-manager dependency and
replacing the velero repository.
It was needed removing because cert-manager already exists
by default in the system avoiding a new install.

Signed-off-by: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
Signed-off-by: Hediberto Cavalcante da Silva <hediberto.cavalcantedasilva@windriver.com>
---
 charts/csm-application-mobility/Chart.yaml | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/charts/csm-application-mobility/Chart.yaml b/charts/csm-application-mobility/Chart.yaml
index f1f6267..71a5ff0 100644
--- a/charts/csm-application-mobility/Chart.yaml
+++ b/charts/csm-application-mobility/Chart.yaml
@@ -13,13 +13,7 @@ version: 0.3.0
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "0.3.0"
-
 dependencies:
-- name: cert-manager
-  version: 1.8.2
-  repository: https://charts.jetstack.io
-  condition: cert-manager.enabled
 - name: velero
   version: 2.29.8
-  repository: https://vmware-tanzu.github.io/helm-charts
-  condition: velero.enabled
+  repository: file://charts/velero
-- 
2.25.1
